File: app/src/main/java/ru/tsu/quizium/MainActivity.java
package ru.tsu.quizium;

import androidx.appcompat.app.AppCompatActivity;

import android.content.Intent;
import android.os.Bundle;
import android.view.View;
import android.widget.Button;
import android.widget.TextView;

import com.google.firebase.auth.FirebaseAuth;

public class MainActivity extends AppCompatActivity {

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);

        Button btn1 = findViewById(R.id.selectQuizButton);
        Button btn2 = findViewById(R.id.rankingButton);
        TextView helloText = findViewById(R.id.helloTextView);

        helloText.setOnClickListener(new View.OnClickListener() {
            @Override
            public void onClick(View v) {
                Intent intent = new Intent(MainActivity.this, AuthorizationActivity.class);
                startActivity(intent);
            }
        });

        btn1.setOnClickListener(new View.OnClickListener() {
            @Override
            public void onClick(View v) {
                Intent intent = new Intent(MainActivity.this, QuizListActivity.class);
                startActivity(intent);
            }
        });

        btn2.setOnClickListener(new View.OnClickListener() {
            @Override
            public void onClick(View v) {
                Intent intent = new Intent(MainActivity.this, QuizResultActivity.class);
                startActivity(intent);
            }
        });
    }
}

File: app/src/main/java/ru/tsu/quizium/MainActivity.kt
package ru.tsu.quizium

import android.content.Intent
import android.os.Bundle
import androidx.appcompat.app.AppCompatActivity
import com.google.firebase.auth.FirebaseAuth
import kotlinx.android.synthetic.main.activity_main.*

class MainActivity : AppCompatActivity() {

    private var mAuth: FirebaseAuth? = null

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main)

        mAuth = FirebaseAuth.getInstance()
        val email = mAuth!!.currentUser?.email
        userEmailTextView.text = email

        changeUserTextView.setOnClickListener {
            val intent = Intent(this@MainActivity, AuthorizationActivity::class.java)
            startActivity(intent)
        }
        selectQuizButton.setOnClickListener {
            val intent = Intent(this@MainActivity, QuizListActivity::class.java)
            startActivity(intent)
        }
        rankingButton.setOnClickListener {
            val intent = Intent(this@MainActivity, QuizResultActivity::class.java)
            startActivity(intent)
        }
    }
}